src/bin/index.js

```diff
@@ -1,8 +1,18 @@
 #!/usr/bin/env node
 
+if (!process.env.DEBUG) {
+  process.env.DEBUG = "phenomic:*"
+}
+
 // still in src/, handy to play
 if (__filename.indexOf("phenomic/src") > 1) {
```

Comment:
I think this is why build failed on Appveyor. This requires the phenomic source folder must be named `phenomic`. Maybe you can try with `indexOf("src/bin/index.js")`